plugin-server/src/cdp/services/hog-watcher.service.ts

```diff
@@ -132,14 +132,37 @@ export class HogWatcherService {
 
             if (result.finished) {
                 // If it is finished we can calculate the score based off of the timings
+                // Separate the duration calculations for 'hog' and 'async_function' kinds
+                let hogDurationMs = 0
+                let asyncDurationMs = 0
+
+                for (const timing of result.invocation.timings) {
+                    if (timing.kind === 'hog') {
+                        hogDurationMs += timing.duration_ms
+                    } else if (timing.kind === 'async_function') {
+                        asyncDurationMs += timing.duration_ms
```

Comment:
I don't think we should be totalling these now that i think about it...

If an individual hog function takes 30ms but due to back and forth with fetch the total time is 120ms we shouldnt be penalising it.

Same with fetch. If one request takes 100ms but within a hog function there were 3 requests we shouldn't be penalising it. So the cost should be _per timing_ not as a total